hack/installer/entrypoint.sh

```diff
@@ -86,14 +87,17 @@ case "$1" in
         d )
           DEVICE=${OPTARG}
           ;;
+        e )
+          EXTRA_ARGS="${EXTRA_ARGS} --extra-kernel-arg=${OPTARG}"
+          ;;
         n )
           TALOS_RAW="/out/${OPTARG}.raw"
           ;;
         p )
           TALOS_PLATFORM=${OPTARG}
           echo "Using kernel parameter talos.platform=${TALOS_PLATFORM}"
           ;;
-       r )
+        r )
```

Comment:
extra space?

Reply:
I think this just aligns with the other cases.

Reply:
^ yeah just a little alignment fix